### DISCUSSION TITLE
도움!

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#305] [질문]
[Biomes! Prehistoric](https://steamcommunity.com/sharedfiles/filedetails/?id=2860715703)

이 모드 알파로 추출되는지 확인해줄 수 있음?

나는 계속 오류 뜨고 추출이 안되가지고

혹시나 내문제인가 싶어서

엑셀파일로 추출되면 공유좀

**Comment by RimWorldKorea** [comment]
확인 중

**Reply by RimWorldKorea** [reply]
<img width="267" alt="화면 캡처 2023-12-18 195724" src="https://github.com/RimWorldKorea/RMK/assets/150018627/c48bd3bd-ff83-41af-b9db-3bc6d03b7f37">

저거 하나 빼고 추출해봐

패치 복잡한것들 가끔 저러는데 하나씩 빼보면서 확인하면 됨

@zilrty

**Reply by zilrty** [reply]
저거 뭔 패치인지 까봐야겠네

저거빼면 추출 되는거지?

ㄳㄳ
